libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/MatrixClient.kt

```diff
@@ -52,8 +53,8 @@ interface MatrixClient {
     val mediaLoader: MatrixMediaLoader
     val sessionCoroutineScope: CoroutineScope
     val ignoredUsersFlow: StateFlow<ImmutableList<UserId>>
+    suspend fun getJoinedRoom(roomId: RoomId): JoinedMatrixRoom?
```

Comment:
Maybe I'd prefer a method on the `BaseRoom` to initialise a `JoinedRoom` from it

```
class BaseRoom {
   ...
    suspend fun asJoined(): JoinedMatrixRoom()? 
}
```

Reply:
We actually need the `RoomListItem` to instantiate the live timeline, so it may not be that easy 🫤 . We could keep a reference to the item in the base room, maybe, or to the `RoomListService`.

Reply:
ok, lets keep it like this then